MinecraftClient/Settings.cs

```diff
@@ -281,12 +281,12 @@ public static void LoadSettings(string settingsfile)
                                                 case "login": Login = argValue; break;
                                                 case "password": Password = argValue; break;
                                                 case "type": AccountType = argValue == "mojang" 
-                                                        ? ProtocolHandler.AccountType.Mojang 
+                                                        ? ProtocolHandler.AccountType.Mojang
```

Comment:
Should probably use `switch` here for readability.

Reply:
Something like what I did now? (Haven't tested yet though)

Reply:
Seems better indeed 👍